README.md

```diff
@@ -387,6 +387,7 @@ In addition to es_config, the following parameters allow the customization of th
 * ```es_max_threads``` the maximum number of threads the process can start. Defaults to 2048 (the minimum required by elasticsearch).
 * ```es_debian_startup_timeout``` how long Debian-family SysV init scripts wait for the service to start, in seconds. Defaults to 10 seconds.
 * ```es_use_repository``` Setting this to `false` will stop Ansible from using the official Elastic package repositories.
```

Comment:
`es_use_repository`  description is now false as it can be used to install elasticsearch package from any repository configured on the system (not only official Elastic repository).

Can you fix the description?

Reply:
Good for you ?